docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -2,36 +2,29 @@
 To extract the configuration metadata from an application/library the _component extractor_ will be used. It extracts the list of all the config interfaces defined in a library or application. The output is a json file `component.config.metadata.json` containing an array of all the components configurations (app configs, pages, components).
 It will also generate the component classes and types metadata from an Otter library/application, outputting them in a json file `component.class.metadata.json`.
 
-Note that the component.class.metadata.json is used at the moment only in rules engine scope, but in the future will be a requirement for the next steps of the Adobe SPA Editor.
 ## How to use it
 
-Since the v3 of Otter, you should use the component-extraction builder directly in your angular.json
+The Component Extractor is accessible via this NgCLI builder: `@o3r/components:extractor`.
 
 First thing to do is to define your given filenames for the configuration in the _package.json_ of the library/app where you run the extractor.
 When running in a library it will use this configuration as the names for the metadata files.
 When running the extractor in an application, it will search for these filenames in each node_module (package.json file) of each library,
-in order to concat the metadata from the file with other libraries metadata and app metadata.
+in order to concat the file's metadata with the metadata of other libraries and the application's metadata.
```

Comment:
The final metadata file generated for an application will be the concatenation of the ones provided by the library used (if available) and the one extracted from the application itself.